front-end/src/Components/Content/OrganizationInformation/ViewEditOrganizationPhone.tsx

```diff
@@ -0,0 +1,174 @@
+import { Fab, Grid, Paper, TextField, Typography } from "@material-ui/core";
+import ClearIcon from "@material-ui/icons/Clear";
+import DoneIcon from "@material-ui/icons/Done";
+import EditIcon from "@material-ui/icons/Edit";
+import React, { Fragment } from "react";
+import { updateOrganizationPhoneDatabase } from "../../../Scripts/firebaseOrganizationUpdates";
+import { PageProps } from "../../Pages";
+
+declare interface ViewEditOrganizationInfoProps {
+  currentOrganizationData: any;
+  setNotification: any;
+  handleLoadOrganizationData: any;
+  setLoadingMessage: any;
+  classes: any;
+}
+
+const ViewEditOrganizationPhone: React.FunctionComponent<ViewEditOrganizationInfoProps> = ({
+  currentOrganizationData,
+  setNotification,
+  handleLoadOrganizationData,
+  setLoadingMessage,
+  classes,
+}) => {
+  const [organizationPhone, setOrganizationPhone] = React.useState<
+    string
+  >(
+    currentOrganizationData?.organizationPhone
+      ? currentOrganizationData.organizationPhone
+      : ""
+  );
+  const [editing, setEditing] = React.useState(!organizationPhone);
+
+  const startEditing = () => {
+    setEditing(true);
+  };
+
+  const cancelEditing = () => {
+    setEditing(false);
+  };
+
+  const cancelEditingOrganizationPhone = () => {
+    cancelEditing();
+    setOrganizationPhone(
+      currentOrganizationData?.organizationPhone
+        ? currentOrganizationData.organizationPhone
+        : ""
+    );
+  };
+
+  const saveOrganizationPhone = () => {
+    if (currentOrganizationData) {
+      setLoadingMessage("Updating Organization Phone...");
+      updateOrganizationPhoneDatabase(
+        currentOrganizationData.userId,
+        organizationPhone
+      )
+        .then((value: any) => {
```

Comment:
You shouldn't need to specify any... See if you can take the any out.

Reply:
Done